app/src/main/java/lt/vilnius/tvarkau/utils/FormatUtils.java

```diff
@@ -13,6 +14,12 @@
 
 public class FormatUtils {
 
+    private static final DateTimeFormatter EXIF_DATE_TIME = DateTimeFormatter.ofPattern("yyyy:MM:dd HH:mm:ss");
+
+    public static String formatExifAsLocalDateTime(@NonNull String exifDateTime) {
```

Comment:
Make this method to return null on exception. From function definition we don't know if it can throw an exception.